Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+gemspec
+gem "logstash", :github => "elasticsearch/logstash", :branch => "1.5"
```

Comment:
This line should be removed as we take this out of the gemspec and the logstash-core gem.